packages/webdriverio/tests/commands/element/moveTo.test.js

```diff
@@ -35,6 +35,21 @@ describe('moveTo', () => {
             .toEqual({ type: 'pointerMove', duration: 0, x: 20, y: 30 })
     })
 
+    it('should do a moveTo with params if getElementRect returned empty object', async () => {
+        const browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+
+        const elem = await browser.$('#elem')
+        request.setMockResponse([{}, { x: 5, y: 10, height: 33, width: 44 }])
```

Comment:
How is this testing the edge case behavior if all values are given? Shouldn't one be omitted?

Reply:
this is the case when getElementRect has not returned any value but execute script returned all the values.

@christian-bromann btw, what should we do in case we have failed to get one of x,y,width or height with JS. Currently it is fallback to 0. But I think we'd better throw an exception.

Can you judge this? :)
@abjerstedt @christian-bromann

Reply:
I think `el.getBoundingClientRect` will always return proper values therefor I would also say we can remove `if (!el || !el.getBoundingClientRect) return`, no?

Reply:
@christian-bromann I have a failing test because of that
`TypeError: el.getBoundingClientRect is not a function`

don't know how to mock that. Anyway it is DOM interaction

Reply:
so if the element isn't found we should throw or are you talking about webdriverio unit test. Also could you rebase the branch to current master?

Reply:
I'm talking about both.

First of all we should throw an exception if getElementRect failed and then getBoundingClientRect failed as well (for whatever reason)

Secondly I have problems with unit test, but I think additional verification is not a problem `if (!el || !el.getBoundingClientRect) return`

Doing rebase, added test for throw error, and will push after it soon

Reply:
@christian-bromann updated